pkgs/development/libraries/gtest/default.nix

```diff
@@ -2,24 +2,19 @@
 
 stdenv.mkDerivation rec {
   pname = "gtest";
-  version = "1.10.0";
+  version = "1.11.0";
 
   outputs = [ "out" "dev" ];
 
   src = fetchFromGitHub {
     owner = "google";
     repo = "googletest";
     rev = "release-${version}";
-    sha256 = "1zbmab9295scgg4z2vclgfgjchfjailjnvzc6f5x9jvlsdi3dpwz";
+    hash = "sha256-SjlJxushfry13RGA7BCjYC9oZqV4z6x8dOiHfl/wpF0=";
   };
 
   patches = [
     ./fix-cmake-config-includedir.patch
-    (fetchpatch {
```

Comment:
Please remove the unused fetchpatch from inputs.

Reply:
Done.